examples/getstarted/src/api/country/content-types/country/schema.json

```diff
@@ -38,23 +38,6 @@
           "localized": true
         }
       }
-    },
-    "description": {
-      "pluginOptions": {
-        "i18n": {
-          "localized": true
-        }
-      },
-      "type": "string",
-      "maxLength": 3
-    },
-    "time": {
-      "pluginOptions": {
-        "i18n": {
-          "localized": true
-        }
-      },
-      "type": "time"
```

Comment:
I guess that change is unrelated?

Reply:
Yes, unrelated :p Those fields were added by mistake a while ago and I would prefer this CT to stay simple/small